src/uu/stat/src/stat.rs

```diff
@@ -110,6 +47,81 @@ pub const F_SIGN: u8 = 1 << 4;
 // unused at present
 pub const F_GROUP: u8 = 1 << 5;
 
+/// checks if the string is within the specified bound
```

Comment:
I was expecting a bit more comment on the profile.
for example, beg isn't commented

Reply:
will do. :). also apologies for the review request antics earlier, was unaware of things :P